roxie/ccd/ccdfile.cpp

```diff
@@ -826,7 +826,7 @@ class IndexCacheWarmer : implements ICacheWarmer
         if (nodeType != NodeNone && !keyFailed && localFile && !keyIndex)
         {
             //Pass false for isTLK - it will be initialised from the index header
-            keyIndex.setown(createKeyIndex(filename, localFile->getCrc(), *localFile.get(), fileIdx, false));
+            keyIndex.setown(createKeyIndex(filename, localFile->getCrc(), *localFile.get(), fileIdx, false, 0));
```

Comment:
this is within warmBlock - is it reading orderd blocks, should it be using a blocked read ahead too here?